tools/ossf_best_practices/gold_criteria.md

```diff
@@ -0,0 +1,152 @@
+> The project MUST have a [reproducible build](https://reproducible-builds.org/). If no building occurs (e.g., scripting languages where the source code is used directly instead of being compiled), select "not applicable" (N/A). (URL required)
+
+_Possible answers: Met/Unmet/NA_
+
+
+## Automated test suite
+
+
+> A test suite MUST be invocable in a standard way for that language.
+
+_Possible answers: Met/Unmet_
+
+> The project MUST implement continuous integration, where new or changed code is frequently integrated into a central code repository and automated tests are run on the result. 
+
+**Met** 
+https://ci.nodejs.org/
+
+> The project MUST have FLOSS automated test suite(s) that provide at least 90% statement coverage if there is at least one FLOSS tool that can measure this criterion in the selected language. 
+
+_Possible answers: Met/Unmet/NA_
+
+> The project MUST have FLOSS automated test suite(s) that provide at least 80% branch coverage if there is at least one FLOSS tool that can measure this criterion in the selected language.
+
+_Possible answers: Met/Unmet/NA_
+
+
+# Security
+
+## Use basic good cryptographic practices
+
+_Note that some software does not need to use cryptographic mechanisms. If your project produces software that (1) includes, activates, or enables encryption functionality, and (2) might be released from the United States (US) to outside the US or to a non-US-citizen, you may be legally required to take a few extra steps. Typically this just involves sending an email. For more information, see the encryption section of [Understanding Open Source Technology & US Export Controls](https://www.linuxfoundation.org/resources/publications/understanding-us-export-controls-with-os-projects/)._
+
+> The software produced by the project MUST support secure protocols for all of its network communications, such as SSHv2 or later, TLS1.2 or later (HTTPS), IPsec, SFTP, and SNMPv3. Insecure protocols such as FTP, HTTP, telnet, SSLv3 or earlier, and SSHv1 MUST be disabled by default, and only enabled if the user specifically configures it. If the software produced by the project does not support network communications, select "not applicable" (N/A).
+
+_Possible answers: Met/Unmet/NA_
+
+
+> The software produced by the project MUST, if it supports or uses TLS, support at least TLS version 1.2. Note that the predecessor of TLS was called SSL. If the software does not use TLS, select "not applicable" (N/A).
+
+_Possible answers: Met/Unmet/NA_
+
+
+
+## Secured delivery against man-in-the-middle (MITM) attacks
+
+> The project website, repository (if accessible via the web), and download site (if separate) MUST include key hardening headers with nonpermissive values. (URL required)
+
+_Possible answers: Met/Unmet_
+
+## Other security issues
+
+> The project MUST have performed a security review within the last 5 years. This review MUST consider the security requirements and security boundary.
+
+_Possible answers: Met/Unmet_
```

Comment:
```suggestion
**Unmet**
```

As far I know, this is unmet, but it is a great initiative to achieve for https://github.com/nodejs/security-wg/issues/846

Reply:
As far as I can tell from the description, this sounds like a thorough review of design and implementation. I wonder how we would get started on that.

> This MAY be done by the project members and/or an independent evaluation. This evaluation MAY be supported by static and dynamic analysis tools, but there also must be human review to identify problems (particularly in design) that tools cannot detect.